cardano-node/src/Cardano/Node/Tracing/Tracers/ChainDB.hs

```diff
@@ -338,7 +353,9 @@ instance MetaTrace  (ChainDB.TraceEvent blk) where
   documentFor _ = Nothing
 
   allNamespaces =
-        map  (nsPrependInner "AddBlockEvent")
+        Namespace [] ["IgnoreBlockOlderThanK"]
```

Comment:
@Jimbo4350 In your latest push you reverted what I modified yesterday. This line should be:

```suggestion
        Namespace [] ["LastShutdownUnclean"]
```